### ISSUE TITLE
Discrete Structures!

### ISSUE THREAD

**Issue by OmidVHeravi** [llSourcell/learn_math_fast#2]
Hey guys, This is one of the best and most comprehensive lists I've ever come across but in my opinion it is missing one very crucial math concept in the entire directory which is Discrete Structures. Let me know if I made a mistake and probably missed it somewhere, however I felt the need to add this as an issue. Plus here's a link to Rafael Pass's cornell class notes on discrete structures. https://www.cs.cornell.edu/~rafael/discmath.pdf

**Comment by BikashPandey17** [comment]
I agree, create a pull request and add it to the list checkout: [contributing.md](https://github.com/llSourcell/learn_math_fast/blob/master/contributing.md)
